drivers/packet/create.go

```diff
@@ -5,15 +5,89 @@
 package packet
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"strings"
+	"time"
 
 	"github.com/drone/autoscaler"
+	"github.com/packethost/packngo"
+	"github.com/rs/zerolog/log"
 )
 
 func (p *provider) Create(ctx context.Context, opts autoscaler.InstanceCreateOpts) (*autoscaler.Instance, error) {
 	p.init.Do(func() {
 		p.setup(ctx)
 	})
 
-	return nil, nil // TODO
+	buf := new(bytes.Buffer)
+	err := p.userdata.Execute(buf, &opts)
+	if err != nil {
+		return nil, err
+	}
+
+	name := strings.ToLower(opts.Name)
+
+	logger := log.Ctx(ctx).With().
+		Str("project", p.project).
+		Str("facility", p.facility).
+		Str("billing", p.billing).
+		Str("plan", p.plan).
+		Str("os", p.os).
+		Str("hostname", p.hostname).
+		Logger()
+
+	cr := packngo.DeviceCreateRequest{
+		HostName:     p.hostname,
+		Facility:     p.facility,
+		Plan:         p.plan,
+		OS:           p.os,
+		ProjectID:    p.project,
+		BillingCycle: p.billing,
+	}
+
+	d, _, err := p.client.Devices.Create(&cr)
+	if err != nil {
+		return nil, err
+	}
+	id := d.ID
+
+	var IPAddress string
+	for i := 0; i < 100; i++ {
```

Comment:
Maybe use the same block as DigitalOcean, e.g. https://github.com/drone/autoscaler/blob/master/drivers/digitalocean/create.go#L79-L114?